Project/GNU/bwfmetaedit.spec

```diff
@@ -11,9 +11,8 @@ Version:		%bwfmetaedit_version
 Release:		1
 Summary:		Supplies technical and tag information about a video or audio file (CLI)
 Group:			Productivity/Multimedia/Other
-License:		GPL
-URL:			http://mediainfo.sourceforge.net/
-Packager:		Jerome Martinez <info@mediaarea.net>
+License:		0BSD
```

Comment:
oops the bad copy paste... True, need to be updated.

Using [`0BSD` classification from SPDX license list](https://spdx.org/licenses/0BSD.html) is a good hint, but we need to be clear about it in the [license](https://github.com/MediaArea/BWFMetaEdit/blob/master/License.html), currently it stipulates "public domain" which is not a legal wording.

[Unlicense](https://spdx.org/licenses/Unlicense.html) is a bit more known but [has some critisim](https://web.archive.org/web/20170301020915/https://lists.opensource.org/pipermail/license-review/2012-January/001386.html) ([summary](https://softwareengineering.stackexchange.com/a/147120)), so `0BSD` may be fine and License text file should be modified accordingly. As it is in public domain we have the right to change the license :).

cc @dericed @kmurmur in case of objection.

Reply:
Sorry for stepping in! Unlicense is indeed a wee bit inconsistent in the current legal ecosystems worldwide, sadly.